webpack.config.js

```diff
@@ -71,7 +72,8 @@ module.exports = {
       new webpack.DefinePlugin({
         "process.browser": true,
         "process.env.NODE_ENV": JSON.stringify(mode),
-        "process.env.COUCH": JSON.stringify(couch)
+          "process.env.COUCH": JSON.stringify(couch),
```

Comment:
If you want to fix the extra spacing here, I wouldn't object. Divya and I are using vscode with the prettier extension to keep all of our javascript in order.